crates/db_schema/src/impls/local_user.rs

```diff
@@ -202,16 +201,6 @@ impl Crud for LocalUser {
       .get_result::<Self>(conn)
       .await?;
 
-    let site_languages = SiteLanguage::read_local_raw(pool).await;
-    if let Ok(langs) = site_languages {
-      // if site exists, init user with site languages
-      LocalUserLanguage::update(pool, langs, local_user_.id).await?;
-    } else {
-      // otherwise, init with all languages (this only happens during tests and
-      // for first admin user, which is created before site)
-      LocalUserLanguage::update(pool, vec![], local_user_.id).await?;
-    }
```

Comment:
Added another change here, this isnt necessary anymore and was causing some confusion.